java/org/apache/coyote/http2/Http2Parser.java

```diff
@@ -327,7 +329,7 @@ protected void readSettingsFrame(int flags, int payloadSize, ByteBuffer buffer)
                 long value = ByteUtil.getFourBytes(setting, 2);
                 Setting key = Setting.valueOf(id);
                 if (key == Setting.UNKNOWN) {
-                    log.warn(sm.getString("connectionSettings.unknown", connectionId, Integer.toString(id),
+                    log.warn(sm.getString("connectionSettings.unknown", rip + "_" + connectionId, Integer.toString(id),
                             Long.toString(value)));
```

Comment:
The right way to add information to a log message is by adding an additional argument to the i18n string.